Formula/fastlane.rb

```diff
@@ -18,7 +18,9 @@ class Fastlane < Formula
     sha256 cellar: :any, mojave:        "42425e2bd0d5322fee4ec9e788bf12822e82c677bb3989cdb48bb004020c588e"
   end
 
-  depends_on "ruby@2.7"
+  # Issue with Ruby 2.7 not finding gems correctly
+  # https://github.com/fastlane/fastlane/issues/18517
+  depends_on "ruby@2.6"
```

Comment:
Do we still need this given your fix in https://github.com/fastlane/fastlane/pull/18520?

Reply:
Also, the `install` script references `2.7` a bit further down, so even if 2.6 is being installed as a dependency I still think that it will actually use 2.7 at runtime:

```
    (bin/"fastlane").write <<~EOS
      #!/bin/bash
      export PATH="#{Formula["ruby@2.7"].opt_bin}:#{libexec}/bin:$PATH"
      export FASTLANE_INSTALLED_VIA_HOMEBREW="true"
      GEM_HOME="#{libexec}" GEM_PATH="#{libexec}" \\
        exec "#{libexec}/bin/fastlane" "$@"
    EOS
```

Reply:
addressed this here: https://github.com/Homebrew/homebrew-core/pull/75161